Checkout/Checkout/Integration/CheckoutAPIServiceIntegrationTests.swift

```diff
@@ -66,9 +66,9 @@ final class CheckoutAPIServiceIntegrationTests: XCTestCase {
 
     // details associated with default apple pay token
     let expectedApplePayDetails = ApplePayDetails(
-      expiryDate: try! CardValidator(environment: .sandbox).validate(expiryMonth: 9, expiryYear: 22).get(),
-      bin: "520424",
-      last4: "6937"
```

Comment:
Unrelated to this PR, but blocker for pipeline. Too many tasks at the moment to separate.